research/mark_generator/training_data_generator.py

```diff
@@ -0,0 +1,77 @@
+import random
+import typing as tp
+from pathlib import Path
+
+import cv2
+import numpy as np
+
+from research.data_augmentator import DataAugmentator
+from research.mark_generator import MarkGenerator, Mark
+
+
+class TrainingDataGenerator:
+    ANSWER_BOXES_PATH = 'data/input/answer_boxes.jpg'
```

Comment:
I don't see these images present anywhere, if the code depends on it the we should add it